job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -731,10 +731,13 @@ class PublisherContext extends AbstractExtensibleContext {
      * @since 1.19
      */
     @RequiresPlugin(id = 'groovy-postbuild')
-    void groovyPostBuild(String script, Behavior behavior = Behavior.DoNothing) {
+    void groovyPostBuild(String script, Behavior behavior = Behavior.DoNothing, boolean useSandbox = false) {
         publisherNodes << new NodeBuilder().'org.jvnet.hudson.plugins.groovypostbuild.GroovyPostbuildRecorder' {
-            delegate.groovyScript(script)
-            delegate.behavior(behavior.value)
+            delegate.script {
```

Comment:
Ah, Ok, I didn't notice this.

It would also possible to only output the new style xml, when that option is actually specified, and otherwise still output the old xml?

Should I implement that?